stack_definition.json

```diff
@@ -500,21 +551,33 @@
     }
   ],
   "outputs": [
+		{
+		  "name": "elasticsearch_hostname",
+		  "value": "ref:./members/4 - Databases for Elasticsearch/outputs/hostname"
+		},
+		{
+		  "name": "elasticsearch_port",
+		  "value": "ref:./members/4 - Databases for Elasticsearch/outputs/port"
+		},
+		{
+		  "name": "elasticsearch_service_credentials_json",
+		  "value": "ref:./members/4 - Databases for Elasticsearch/outputs/service_credentials_json"
+		},
```

Comment:
We also need Elastic instance CRN (`crn` from Elastic DA) as output. It's needed by the App Config.